matching-notification-rest/src/main/java/org/phenotips/matchingnotification/rest/MatchingNotificationResource.java

```diff
@@ -239,4 +239,18 @@ Response setUserContacted(@FormParam("matchesIds") Set<Long> matchesIds,
     @Produces(MediaType.APPLICATION_JSON)
     @Path("/save-note")
     Response addNote(@FormParam("matchesIds") Set<Long> matchesIds, @FormParam("note") String note);
+
+    /**
+     * Computes phenotype similarity score.
+     *
+     * @param referencePatient JSON
+     * @param matchPatient JSON
+     * @return result JSON
+     */
+    @POST
+    @Consumes(MediaType.APPLICATION_JSON)
```

Comment:
This doesn't really consume json, since it manually parses jsons from an x-www-form-encoded list of parameters.